### ISSUE TITLE
The TabView's close tab button's ╳ becomes clipped when text scaling >=175%.

### ISSUE THREAD

**Issue by kmgallahan** [microsoft/microsoft-ui-xaml#1650]
**Describe the bug**

The TabView's close button's ╳ character becomes clipped when text scaling >=175%.

**Related**: The add new tab button's 🞡 character seems to be scaling as well, but it isn't getting clipped at least.

**Steps to reproduce the bug**

1. Set text scaling to 175% or greater
2. Open the XAML Controls Gallery
3. View the TabView example

**Expected behavior**

The ╳ should stay the same size and remain visible.

It should have `IsTextScaleFactorEnabled="false"`.

**Screenshots**
![Annotation 2019-11-20 191600](https://user-images.githubusercontent.com/5379488/69289641-2f6a8c80-0bcb-11ea-8d02-95ced961e05f.png)

**Version Info**
XAML Controls Gallery version 1.2.11.0

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2019 Update (18362)            | Yes |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Comment by chingucoding** [comment]
I agree, the tab close button should not scale. Should the + button scale?

Without + button scaling:
![image](https://user-images.githubusercontent.com/16122379/69344216-4b276e80-0c6f-11ea-827f-4a537e233765.png)

With + button scaling:
![image](https://user-images.githubusercontent.com/16122379/69344280-68f4d380-0c6f-11ea-9894-97050b426301.png)

All buttons scaling:
![image](https://user-images.githubusercontent.com/16122379/69344665-0b14bb80-0c70-11ea-9bea-45c0a6cebca6.png)

**Comment by kmgallahan** [comment]
> I agree, the tab close button should not scale. Should the + button scale?

I think this aligns with the question of what chevrons should be scaled presented in #1519 

The question is: when is it appropriate to be scaling these text symbol icons vs. when is it not?

In some cases it just looks better having them scaled next to text as seen above, but then the tab icons would also probably look better is they were scaled up too, and thus things get complicated...

**Comment by jevansaks** [comment]
I think the default answer is: don't scale icons, even if they're glyph-based. The user wanted text to be bigger but if we made icons bigger too it would be the same behavior as DPI scaling for the most part. The spirit of the setting is that just text gets bigger.

So the fix for these would be to set IsTextScaleFactorEnabled=false on both + and x.
